app/src/app/list/polls-table/polls-table.component.sass

```diff
@@ -44,3 +46,44 @@ table
         cursor: pointer
         &:hover
           color: $maincolor
+
+@media screen and (max-width: $breakpoint-tablet) 
+  table 
+    border: 0
+    thead
+      clip: rect(0 0 0 0)
+      height: 1px
+      margin: -1px
+      overflow: hidden
+      padding: 0
+      position: absolute
+    tr 
+      border: solid 1px $maincolor
+      display: block
+      margin-bottom: 0.625em
+      padding: 0.35em
+      width: 15em
```

Comment:
To hide the rest of the characters if the title is too long, so I suppose the fixed-width...
![image](https://user-images.githubusercontent.com/72395200/120930680-a05e9000-c6ee-11eb-884c-825cf2091977.png)

Reply:
This is because the table element still has `display: table` and the tbody element `display: table-row-group`. When both have `display: block` there is no such issue.